BuildArchive.sh

```diff
@@ -26,12 +26,19 @@ make -j $((`nproc` + 2)) tmp/hoot-deps
 
 # init and start Postgres
 cd /tmp
-PG_VERSION=9.2
-sudo service postgresql-$PG_VERSION initdb
-sudo service postgresql-$PG_VERSION start
+# init and start Postgres
+PG_SERVICE=$(ls /etc/init.d | grep postgresql- | sort | tail -1)
```

Comment:
I don't think so. Postgres has a version number on Centos and it doesn't on Ubuntu.